core/src/main/java/org/apache/iceberg/puffin/StandardBlobTypes.java

```diff
@@ -26,4 +26,17 @@ private StandardBlobTypes() {}
    * href="https://datasketches.apache.org/">Apache DataSketches</a> library
    */
   public static final String APACHE_DATASKETCHES_THETA_V1 = "apache-datasketches-theta-v1";
+
+  /**
+   * A serialized form of <a
+   * href="https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/stats/StatsUtils.java#L854C17">
+   * HIVE_COLUMN_STATS_OBJ</a>
+   */
+  public static final String HIVE_COLUMN_STATS_OBJ = "column-statistics-obj";
```

Comment:
Nothing prevents Hive from using arbitrary String IDs for Hive-specific statistics, however I'd encourage to prefix Hive-specific statistics with eg "hive." to avoid conflicts in the future.

If this is supposed to be a "standard blob type", it needs to be documented at https://iceberg.apache.org/puffin-spec/#blob-types in Hive-agnostic way

Reply:
Corresponding iceberg-docs PR : https://github.com/apache/iceberg-docs/pull/269